### DISCUSSION TITLE
Error: Unable to load application: ActiveRecord::ConnectionNotEstablished

### DISCUSSION THREAD

**Discussion by danpardella** [mastodon/mastodon#21946] [Q&A]
I've restarted the cloud server for my mastodon instance and the system does not come back to live...

I checked with: 

`journalctl -u mastodon-web`

and get this back. So here is an error with the Postgres? I can't find any further information on my search...

can anybody help?

Infos: 
- not an docker installation
- on Hetzner cloud server
- wanted to delete media files with `tootctl` -> but system can't find. worked an that issue with some tips from the web...doesn't work...
- after that this error is here...

`[5386] ! Unable to load application: ActiveRecord::ConnectionNotEstablished: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: No such file or directory`
`Dec 02 12:33:52 master-webworker-social bundle[5386]:         Is the server running locally and accepting connections on that socket?`
`Dec 02 12:33:52 master-webworker-social bundle[5386]: bundler: failed to load command: puma (/home/mastodon/live/vendor/bundle/ruby/3.0.0/bin/puma)`
`Dec 02 12:33:52 master-webworker-social bundle[5386]: /home/mastodon/live/vendor/bundle/ruby/3.0.0/gems/activerecord-6.1.7/lib/active_record/connection_adapters/postgresql_adapter.rb:83:in 'rescue in new_client': connection to server on socket "/var/run>`
`Dec 02 12:33:52 master-webworker-social bundle[5386]:         Is the server running locally and accepting connections on that socket?`
`Dec 02 12:33:52 master-webworker-social bundle[5386]:         from /home/mastodon/live/vendor/bundle/ruby/3.0.0/gems/activerecord-6.1.7/lib/active_record/connection_adapters/postgresql_adapter.rb:77:in 'new_client'`

**Comment by mohe2015** [comment]
what does `sudo systemctl status postgresql` or `sudo systemctl status postgres` say? is it green?
